src/main/java/ua/softserveinc/tc/controller/util/ViewEventController.java

```diff
@@ -148,9 +150,13 @@ public String gerRecurrent(@RequestBody RecurrentEventDto recurrentEventDto, Bin
     }
 
     @PostMapping(value = "getmonthlyevents", produces = "application/json")
-    @ResponseBody
```

Comment:
as i know responseBody makes ResponseEntity from the Json obj, so if i send ResponseEntity directly i don't need that annotation. I may be mistaken..